testing/test_pytest_html.py

```diff
@@ -418,6 +418,39 @@ def test_fail():
             assert link in html
             assert os.path.exists(src)
 
+    @pytest.mark.parametrize('src_type', ["https://", "file://"])
+    def test_extra_image_non_b64(self, testdir, src_type):
```

Comment:
For coverage, we should also include a test that creates a file in the testdir and references it as an image path.